content/_data/changelogs/weekly.yml

```diff
@@ -8542,6 +8542,64 @@
 
   # pull: 4875 (PR title: Add new line at EOF in config.xml)
 
+- version: '2.251'
+  date: 2020-08-05
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4892
+    issue: 63276
+    authors:
+    - fqueiruga
+    message: |-
+      Empty installed plugins table text is readable again (regression in 2.249).
+  - type: major bug
+    category: regression
+    pull: 4870
+    issue: 63180
+    authors:
+    - fqueiruga
+    message: |-
+      Restore wrapping tabs into multiple lines instead of overflowing (regression in 2.248).
+  - type: rfe
+    category: rfe
```

Comment:
This could be a bug, colours were inconsistent...

Reply:
Agreed.  Updated in 452bcc16d1ccb8cf011c8e2f4deb32df68981f3e